OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -35,6 +35,14 @@ public enum FlightDynamic
 		TakeOffOnCreation = 128,
 	}
 
+	public enum IdleBehaviorType
+	{
+		None, // Aircraft will hover or fly in circles, depending on FlightDynamics. This is also the fallback if one of the below fails.
```

Comment:
This is already documented in the `Desc`, do we want to duplicate that here? (Imho no, because there is a chance both will get out of sync, and it is duplicated information anyway.)

Reply:
Makes sense. Can we please remove this?